packages/api/test/__fixtures__/sdk/readme/types.ts

```diff
@@ -0,0 +1,173 @@
+import type { FromSchema } from 'json-schema-to-ts';
```

Comment:
You might (not) notice it but there are types being exported here that aren't actually used in the codegen'd SDK and that's because the way we generate JSON Schema in `oas` is to process all `components`. Because we're supplying the custom `transformer` to JSON Schema generation there in order to rewrite their names to `::convert::SCHEMA_NAME` so we can later turn it into a literal, this transformer work is running on all components regardless if they're being used or not.

I'm okay with this! Even if the type isn't used in the codegen'd SDK making it available to folks seems doesn't seem like a problem.